test/fixtures/output/environment/11-prompt.golden

```diff
@@ -1,6 +1,6 @@
 Are you sure you want to delete environment "not-595"?
 To confirm, type "default". To cancel, press Ctrl-C: Deleted environment "not-595".
-this command requires an environment; no environments found
+WARNING: no environment found
```

Comment:
Shouldn't this be an error and not a warning? Doesn't the operation fail?

Reply:
The function w/ this ultimately gets called in dozens of files, so I'm not completely sure if there are any which can continue or if all of them invariably fail without this id.

This function returns only a string and no error, though, so I can start working on a followup PR to convert this to an error and add error handling to its calls but we should probably merge without it and this can go in next week's release.

Reply:
I'm guessing that 99% of the time this should be an error and not a warning. Returning a `(string, error)` is definitely the way to go here (but it'll be a decent amount of work)!